### DISCUSSION TITLE
韓國擬試行AI臉部辨識 追蹤COVID-19病例

### DISCUSSION THREAD

**Discussion by jacky0115** [1101-GS3073-C/Python-Community-GS3073-C#189] [科技新知]
[韓國擬試行AI臉部辨識 追蹤COVID-19病例](https://www.cna.com.tw/news/aopl/202112130157.aspx)
韓國利用AI演算法和臉部辨識技術，分析超過1萬820台閉路電視攝影機蒐集到的畫面，追蹤確診者及和他們近距離接觸者的足跡，並且分析是否有戴口罩，然而此舉引發侵害隱私的疑慮

**Comment by briankuo123** [comment]
這感覺跟中國做的事情一模一樣，難怪他們會覺得隱私受到侵犯。

**Comment by minniewei** [comment]
我覺得這麼做壞處大於好處....會不會新冠只是障眼法，其實是想學中國...?

**Comment by Timmy110403540** [comment]
在防疫的同時，人們的隱私可能會不被重視。

**Comment by wh20502** [comment]
我覺得好像已經來不及了，https://udn.com/news/story/6811/5957630南韓已經開始行動，至少在台灣不會這樣就好了。

**Comment by Shiba010** [comment]
這有點類似中國的天眼系統，感覺有隱私方面的疑慮。

**Comment by Lynn12378** [comment]
雖然個人隱私好像有點疑慮，但是防疫感覺還是必較重要的

**Comment by LanningYang** [comment]
人臉辨識雖有隱私疑慮 但個人覺得還是要以多數人的利益為考量

**Comment by ghost** [comment]
可是總覺得隱私跟安全 安全比較重要

**Comment by lx3211** [comment]
現在疫情已經緩解了許多，還好台灣沒有要推廣這個？

**Comment by NcuJerryKao** [comment]
隱私問題 讓我想到那稜鏡計劃
或許(試行AI臉部辨識 追蹤COVID-19病例)的隱私問題 也只是提出來 不論人類的反應 仍會進行下去

**Comment by number32984** [comment]
利用AI演算來檢視大家防疫的同時也必須顧慮到其會產生的負面效果

**Comment by Ben6uo** [comment]
不是共存了嗎 為什麼還要這樣做 好可怕

**Comment by BingRong1217** [comment]
是否會產生隱私權問題

**Comment by skyling0299** [comment]
很像天眼計畫，覺得不優，侵犯人的隱私權

**Comment by limitx0** [comment]
韓國那邊不知道有沒有人抗議

**Comment by Aerofe** [comment]
這樣感覺會讓人民都沒有隱私

**Comment by awehao** [comment]
應該不是真的為了防疫拉

**Comment by Chenszuhong** [comment]
我也覺得用AI去辨識人臉會影響個資安全 感覺很像去每個地方都被追蹤

**Comment by Raywei0214** [comment]
問題的關鍵究竟為何？若無法徹底理解人臉辨識，恐怕會是人類的一大遺憾。
我們都有個共識，若問題很困難，那就勢必不好解決。

**Comment by gnsJhenJie** [comment]
我覺得用手機藍芽去識別化的追蹤就很夠了(像 台灣社交距離那個App一樣)，尤其是在南韓智慧型手機這麼普遍的狀況下，若是單純做研究還可以接受，大規模使用就不太合理了。

**Comment by bww0401** [comment]
雖然會有去哪都被跟蹤的感覺，但如果真的是為了防疫，我可以給過ㄟ

**Comment by larry40305** [comment]
現在中國很需要這類產品，尤其現在開放後

**Comment by Xuewen628** [comment]
我想知道若用於人臉辨識 國家單位可以給出怎麼樣的保證來確保隱私問題
畢竟若各資外流會給國民帶來很多的麻煩
雖然這個機制是初期制度
但還是需要經過深思熟慮
畢竟影響的範圍是全國國民

**Comment by boson13579** [comment]
想要同時做好防疫追蹤跟顧好人民隱私很難，兩者本身就互相衝突。個人偏好尊重個人隱私，畢竟這種傳染力極強的疾病致死率通常不會太高，致死率高的疾病傳染力一般來說也不會太強，所以沒必要為了致死率不高的疾病破壞人民生來就有的隱私權。
